ArkHomes/ArkHomes/ArkHomes.vcxproj

```diff
@@ -46,14 +46,15 @@
       <PreprocessorDefinitions>NDEBUG;ARKHOMES_EXPORTS;_WINDOWS;_USRDLL;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <ConformanceMode>true</ConformanceMode>
       <LanguageStandard>stdcpp17</LanguageStandard>
-      <AdditionalIncludeDirectories>..\Includes;\..\..\ARK-Server-API\version\Core\Public;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>..\Includes;..\..\..\ARK-Server-API\version\Core\Public;..\..\Permissions\Permissions\Public;..\..\ArkShop\ArkShop\Public;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
     </ClCompile>
     <Link>
       <SubSystem>Windows</SubSystem>
       <EnableCOMDATFolding>true</EnableCOMDATFolding>
       <OptimizeReferences>true</OptimizeReferences>
       <GenerateDebugInformation>true</GenerateDebugInformation>
       <AdditionalLibraryDirectories>\..\..\ARK-Server-API\out_lib;\..\Ark-Server-Plugins\Permissions\out_lib;\..\ArkShop\out_lib;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
```

Comment:
The "AdditionalLibraryDirectories" is listed twice now with the old and new updates.

Reply:
Ok, thanks.